crates/shielder-relayer/src/price_feed/mod.rs

```diff
@@ -82,17 +116,13 @@ impl Prices {
                 .expect("Mutex poisoned")
                 .replace(price);
         }
-
-        Ok(())
     }
 }
 
 /// Start a price feed that updates the prices in the given `Prices` instance.
 pub async fn start_price_feed(prices: Prices) -> Result<(), anyhow::Error> {
```

Comment:
can change the output type to `!`

Reply:
unfortunately I can't - it is used within `tokio::try_join!`, which requires `Result`